sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql

```diff
@@ -46,6 +46,8 @@ store_stats AS (
   ON
     country_names.name = views_data.country_name
 ),
+-- TODO: we may need to consider updating the source for new profiles to be firefox_ios.firefox_ios_clients
```

Comment:
@soGaussian what are your thoughts here? I think it would make sense to use use an upstream table instead to make sure that at least this subset of tables is aligned.